tribe-ext-link-events-to-website-url/index.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Plugin Name:     The Events Calendar: Link Event Titles to Event Website URLs
+ * Description:     An extension that makes event titles link to the events' Website URLs when present.
+ * Version:         1.0.0
+ * Extension Class: Tribe__Extension__Link_Event_Titles_to_Event_Website_URLs
+ * Author:          Modern Tribe, Inc.
+ * Author URI:      http://m.tri.be/1971
+ * License:         GPLv2 or later
+ * License URI:     https://www.gnu.org/licenses/gpl-2.0.html
+ */
+
+// Do not load directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+
+// Do not load unless Tribe Common is fully loaded.
+if ( ! class_exists( 'Tribe__Extension' ) ) {
+	return;
+}
+
+class Tribe__Extension__Link_Event_Titles_to_Event_Website_URLs extends Tribe__Extension {
+
+    /**
+     * Setup the Extension's properties.
+     *
+     */
+    public function construct() {
+        $this->add_required_plugin( 'Tribe__Events__Main', '4.3.3' );
+        $this->set_url( 'https://theeventscalendar.com/extensions/make-event-titles-link-to-the-event-website-url/' );
+    }
+
+    /**
+     * Extension initialization and hooks.
+     */
+    public function init() {
+    	add_filter( 'tribe_get_event_link', array( $this, 'make_event_titles_link_to_event_website' ), 100, 2 );
+    }
+
+	/**
+	 * Make event titles link to URLs from their respective "Event Website" fields.
+	 *
+	 * @param string $link
+	 * @param int $post_id
+	 * @return string
+	 */
+	public function make_event_titles_link_to_event_website( $link, $post_id ) {
```

Comment:
This too would be nice to shorten. In a lot of our classes we name the methods after the filter they are attached to. We have a number of classes with a 'pre_get_posts()` method for instance, that is always attached to `pre_get_posts()`. This would be a great trend to continue with extensions, particularly because it allows us to easily merge extensions into existing methods, if they ever graduate to being built-in features. So maybe the name `tribe_get_event_link` would be appropriate?